Cargo.lock

```diff
@@ -1131,8 +1131,10 @@ dependencies = [
  "substrate-client 1.0.0 (git+https://github.com/joystream/substrate.git?rev=6dfc3e8b057bb00322136251a0f10305fbb1ad8f)",
  "substrate-consensus-aura-primitives 1.0.0 (git+https://github.com/joystream/substrate.git?rev=6dfc3e8b057bb00322136251a0f10305fbb1ad8f)",
  "substrate-consensus-authorities 1.0.0 (git+https://github.com/joystream/substrate.git?rev=6dfc3e8b057bb00322136251a0f10305fbb1ad8f)",
+ "substrate-forum-module 0.1.0 (git+https://github.com/bedeho/substrate-forum-module?rev=9c0cf6544f8f6f9b7ae185007ccf3d8623d5f62a)",
```

Comment:
This doesn't match with: https://github.com/Joystream/substrate-runtime-joystream/pull/66/files#diff-80398c5faae3c069e4e6aa2ed11b28c0R196

Can we update after merging the runtime integration PR